src/sentry/models/auditlogentry.py

```diff
@@ -260,6 +264,16 @@ def get_note(self):
             return 'requested to transfer project %s' % (self.data['slug'], )
         elif self.event == AuditLogEntryEvent.PROJECT_ACCEPT_TRANSFER:
             return 'accepted transfer of project %s' % (self.data['slug'], )
+        elif self.event == AuditLogEntryEvent.PROJECT_ENABLE:
+            if isinstance(self.data['state'], set):
+                return 'enabled project filter %s' % (', '.join(self.data["state"]),)
+            else:
+                return 'enabled project filter %s' % (self.data['state'], )
+        elif self.event == AuditLogEntryEvent.PROJECT_DISABLE:
+            if isinstance(self.data['state'], set):
```

Comment:
Do you need to handle lists or tuples?

Reply:
Lists

Reply:
Should this join case also be run for lists then? If your check was

```python
if isinstance(self.data['state'], six.string_types):
   return 'disabled project filter %s' % (self.data['state'], )
return 'disabled project filter %s' % (', '.join(self.data["state"]),)
```
You could handle strings, lists and other enumerables with fewer typechecks.